samples/app-test/android-app-test.js

```diff
@@ -0,0 +1,79 @@
+import 'babel-polyfill'
+import 'colors'
+import wd from 'wd'
+import {assert} from 'chai'
+
+const username = process.env.KOBITON_USERNAME
+const apiKey = process.env.KOBITON_API_KEY
+
+const kobitonServerConfig = {
+  protocol: 'https',
+  host: 'api.kobiton.com',
+  auth: `${username}:${apiKey}`
+}
+
+const desiredCaps = {
+  sessionName:        'Automation test session',
+  sessionDescription: '', 
+  deviceOrientation:  'portrait',  
+  captureScreenshots: true, 
+  app:                '<APP_URL>', 
+  deviceGroup:        'KOBITON', 
+  deviceName:         '<KOBITON_DEVICE_NAME>',
+  platformVersion:    '<KOBITON_DEVICE_VERSION>',
+  platformName:       'Android' 
+}
+
+desiredCaps.app = 'https://appium.github.io/appium/assets/ApiDemos-debug.apk'
```

Comment:
Please replace this line to the object above

Reply:
@NguyenHoanTien This script is only used for testing `ApiDemos-debug` app. If the end-users paste the desiredCaps from Kobiton portal, they will have to edit the `app` element to this url. I did this so that they only need to replace the existed desiredCaps with the one they got from Kobiton, and then the script will do the rest.